release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -2988,7 +2988,9 @@ The default `IngressController` CR is configured with 4 HAProxy threads. If you
 
 * There is a known issue that prevents installing a cluster on or updating a cluster to this version of {product-title} on Microsoft Azure Stack Hub. For more details and a workaround, see the information in this link:https://access.redhat.com/solutions/7040264[Red Hat Knowledgebase article]. (link:https://issues.redhat.com/browse/OCPBUGS-20548[*OCPBUGS-20548*])
 
-* There is a known issue with Microsoft Azure clusters that use Azure AD Workload Identity. A recent change to the default security settings for new Azure storage accounts in the `eastus` region prevents the installation of clusters that use Azure AD Workload Identity in that region. Other regions do not seem to be impacted at this time, but might be impacted in the future.
+* There is a known issue with Microsoft Azure clusters that use Azure AD Workload Identity in versions of {product-title} {product-version} prior to version 4.14.2. A recent change to the default security settings for new Azure storage accounts in the `eastus` region prevents the installation of clusters that use Azure AD Workload Identity in that region. Other regions do not seem to be impacted at this time, but might be impacted in the future.
++
+This issue is resolved in {product-title} xref:../release_notes/ocp-4-14-release-notes.adoc#ocp-4-14-2[4.14.2].
```

Comment:
Just so I understand, this goes in the Known Issues section of the 4.14.2 release notes? @jeana-redhat

Reply:
The known issue text from GA should be updated just in the same place as done in this PR. It's still a known issue in 4.14.0, so the workaround is relevant to anyone who has not updated to a version in which it is fixed. The change is so that anyone on 4.14.0-.1 knows they can get the fix in 4.14.2.

The only addition to 4.14.2 specifically is [the line](https://github.com/openshift/openshift-docs/pull/67375/files#diff-015f97d3990eab372e83b83257100cc8db66502abdb4c46c227409914734503aR3107) containing the bugfix text